cli/util/utf8.js

```diff
@@ -42,38 +33,37 @@ utf8.length = function utf8_length(string) {
  * @param {number} end Source end
  * @returns {string} String read
  */
-utf8.read = function utf8_read(buffer, start, end) {
+export function read(buffer, start, end) {
   var len = end - start;
-  if (len < 1)
-    return "";
+  if (len < 1) return "";
   var parts = null,
       chunk = [],
       i = 0, // char offset
       t;     // temporary
   while (start < end) {
     t = buffer[start++];
-    if (t < 128)
+    if (t < 128) {
       chunk[i++] = t;
-    else if (t > 191 && t < 224)
+    } else if (t > 191 && t < 224) {
       chunk[i++] = (t & 31) << 6 | buffer[start++] & 63;
-    else if (t > 239 && t < 365) {
+    } else if (t > 239 && t < 365) {
       t = ((t & 7) << 18 | (buffer[start++] & 63) << 12 | (buffer[start++] & 63) << 6 | buffer[start++] & 63) - 0x10000;
       chunk[i++] = 0xD800 + (t >> 10);
       chunk[i++] = 0xDC00 + (t & 1023);
-    } else
+    } else {
       chunk[i++] = (t & 15) << 12 | (buffer[start++] & 63) << 6 | buffer[start++] & 63;
-    if (i > 8191) {
+    }
+    if (i >= 8192) {
```

Comment:
```suggestion
    if (i >= 0x2000) {
```

Reply:
This is just a size/length, not a value, so would rather leave it as-is.